### PR TITLE
Add support for JMX monitoring in Artifactory

### LINKED ISSUE #157: [Feature] Enable JMX monitoring
Would it be possible to enable the chart to expose the JMX monitoring port via a value option for use with Prometheus's JMX exporter

### PR DESCRIPTION
**Pull request by danielezer**
#### PR Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [x] CHANGELOG.md updated
- [x] Variables and other changes are documented in the README.md

**What this PR does / why we need it**:
Add support for JMX monitoring with Artifactory

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #
fixes #157 

**Special notes for your reviewer**:
This enabled basic support for JMX monitoring. We can consider adding support for authentication later

### CODE REVIEW

File: stable/artifactory-ha/README.md [hunk 1]
```diff
@@ -328,7 +331,10 @@ to your choice of port
 In order to connect to Artifactory using JMX with jconsole (or any similar tool) installed on your computer, follow the following steps:
 1. Enable JMX as described above and Change the Artifactory service to be of type LoadBalancer:
 ```bash
-helm install --name artifactory jfrog/artifactory --set artifactory.primary.javaOpts.jmx.enabled=true artifactory.node.javaOpts.jmx.enabled=true --set artifactory.service.type=LoadBalancer
+helm install --name artifactory \
+    --set artifactory.primary.javaOpts.jmx.enabled=true \
+    artifactory.node.javaOpts.jmx.enabled=true \
```
Comment (eldada) [inline stable/artifactory-ha/README.md]: Missing `--set`

**Comment by danielezer** [comment]
CI works on gke with
```make gke -- --charts stable/artifactory,stable/artifactory-ha```

**Comment by rimusz** [comment]
artifactoty/artifactory-ha both have a few tests to run for CI, having both in the same PR, travisCI timeouts as it takes too long to run all these tests

**Comment by danielezer** [comment]
Thanks @rimusz :)

**Pull request merged**